src/modules/blockchainApplication/explore/components/BlockchainAppDetailsHeader/index.js

```diff
@@ -44,21 +44,23 @@ const BlockchainAppDetailsHeader = ({ application, chainAction, loading }) => {
               <span className="chain-name-text">{name}</span>
               {chainAction}
             </div>
-            <Box className={styles.addressRow}>
-              <ValueAndLabel className={styles.transactionId}>
-                <span className="copy-address-wrapper">
-                  <CopyToClipboard
-                    text={address}
-                    value={address}
-                    className="tx-id"
-                    containerProps={{
-                      size: 'xs',
-                      className: 'copy-address',
-                    }}
-                  />
-                </span>
-              </ValueAndLabel>
-            </Box>
+            {!!address && (
```

Comment:
Any reason why we are not just doing `address && (...~

Reply:
@oskarleonard Slight oversight while implementing that but I just fixed it. Good catch